package protoFile.hall.quitHallMessage;
message QuitHallMessage{
     required int32 signal = 1;
}